Test case failed occasionally in `test/unit/expire.tcl`

### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-kvrocks/issues) and found no similar issues.


### Motivation

Check expire at test case failed occasionally, we should fix it to make the CI happy.

```
*** [err]: EXPIREAT - Check for EXPIRE alike behavior in tests/unit/expire.tcl
Expected '1[345]' to equal or match '16'
Error: Process completed with exit code 1.
```

### Solution

_No response_

### Are you willing to submit a PR?

- [X] I'm willing to submit a PR!

--- a/tests/tcl/tests/unit/expire.tcl
+++ b/tests/tcl/tests/unit/expire.tcl
@@ -86,9 +86,9 @@ start_server {tags {"expire"}} {
 
     test {PERSIST can undo an EXPIRE} {
         r set x foo
-        r expire x 50
+        r expire x 12
         list [r ttl x] [r persist x] [r ttl x] [r get x]
-    } {50 1 -1 foo}
+    } {1[012] 1 -1 foo}
 
     test {PERSIST returns 0 against non existing or non volatile keys} {
         r set x foo

--- a/tests/tcl/tests/unit/expire.tcl
+++ b/tests/tcl/tests/unit/expire.tcl
@@ -56,7 +56,7 @@ start_server {tags {"expire"}} {
         r set x foo
         r expireat x [expr [clock seconds]+15]
         r ttl x
-    } {1[345]}
+    } {1[3456]}
 
     test {SETEX - Set + Expire combo operation. Check for TTL} {
         r setex x 12 test

